lib/request.js

```diff
@@ -83,7 +83,7 @@ class Request {
 
         // If the API limit has been reached
         if (res.statusCode === 429) {
-          const timeToWait = res.headers['x-retry-after'];
+          const timeToWait = res.headers['X-Rate-Limit-Time-Reset-Ms'];
```

Comment:
resource for future devs: https://developer.bigcommerce.com/api-docs/getting-started/best-practices#api-rate-limits